src/main/kotlin/no/nav/brukerdialog/ytelse/pleiepengersyktbarn/søknad/api/domene/k9Format/K9Arbeidstid.kt

```diff
@@ -3,31 +3,41 @@ package no.nav.brukerdialog.ytelse.pleiepengersyktbarn.søknad.api.domene.k9Form
 import no.nav.brukerdialog.ytelse.pleiepengersyktbarn.søknad.api.domene.Arbeidsgiver
 import no.nav.brukerdialog.ytelse.pleiepengersyktbarn.søknad.api.domene.PleiepengerSyktBarnSøknad
 import no.nav.k9.søknad.felles.type.Organisasjonsnummer
+import no.nav.k9.søknad.felles.type.Periode
 import no.nav.k9.søknad.ytelse.psb.v1.arbeidstid.Arbeidstaker
 import no.nav.k9.søknad.ytelse.psb.v1.arbeidstid.Arbeidstid
-import java.time.LocalDate
 
 internal fun PleiepengerSyktBarnSøknad.byggK9Arbeidstid(): Arbeidstid {
+    val søknadsperiode = Periode(fraOgMed, tilOgMed)
     val arbeidstid = Arbeidstid().apply {
 
-        if(arbeidsgivere.isNotEmpty()) medArbeidstaker(arbeidsgivere.tilK9Arbeidstaker(fraOgMed, tilOgMed))
+        if (arbeidsgivere.isNotEmpty()) medArbeidstaker(arbeidsgivere.tilK9Arbeidstaker(søknadsperiode))
+
+        val frilansArbeidstidInfo = frilans.k9ArbeidstidInfo(søknadsperiode)
+        val omsorgsstønadArbeidstidInfo = omsorgsstønad?.k9ArbeidstidInfo(søknadsperiode)
+
+        medFrilanserArbeidstid(
+            ArbeidstidInfoUtleder(
+                arbeidstidInfoer = listOf(
+                    frilansArbeidstidInfo,
+                    omsorgsstønadArbeidstidInfo ?: frilansArbeidstidInfo
```

Comment:
Dette blir vel litt rart? Dersom omsorgsstønadArbeidstidInfo finnes så blir vel faktisk og normal arbeidstid doblet siden du summerer to like tidslinjer.

Reply:
Den feil etter refaktorering.
Fikser 👍